src/tree_data.h

```diff
@@ -1168,7 +1167,6 @@ LY_ERR lyd_insert_before(struct lyd_node *sibling, struct lyd_node *node);
  * @brief Insert a node after another node, can be used only for user-ordered nodes.
  *
  * - if the node is part of some other tree, it is automatically unlinked.
```

Comment:
please say explicitly: `So in case of inserting a list of nodes, they must be inserted one by one.` and the same in `lyd_insert_before()`

Reply:
Fixed.